tools/cli/adminElasticSearchCommands.go

```diff
@@ -202,6 +205,59 @@ func AdminIndex(c *cli.Context) {
 	}
 }
 
+func AdminDelete(c *cli.Context) {
+	esClient := getESClient(c)
+	indexName := getRequiredOption(c, FlagIndex)
+	inputFileName := getRequiredOption(c, FlagInputFile)
+	batchSize := c.Int(FlagBatchSize)
+	rps := c.Int(FlagRPS)
+	ratelimiter := tokenbucket.New(rps, clock.NewRealTimeSource())
+
+	file, err := os.Open(inputFileName)
+	if err != nil {
+		ErrorAndExit("Cannot open input file", nil)
+	}
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+
+	scanner.Scan() // skip first line
```

Comment:
what is the first line and why it has to skip?

Reply:
added in helper message. First line is list result table header, see top comments for use example